public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
+
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference to the older version to compare
+            (defaults to HEAD)
+  b_ref     Git reference to the newer version to compare
+            (defaults to the current workspace including changes)
+```
+
+## Description
+
+The changes shown by this command includes the new value, and numeric difference
+(delta) from the previous value of metrics (with 3-digit accuracy). They're
+calculated between two commit SHA hashes, branch or tag names, etc.
+([Git references](https://git-scm.com/docs/revisions)) for all metrics in the
+<abbr>project</abbr>, found by examining all of the
+[DVC-files](/doc/user-guide/dvc-file-format) in both references.
+
+The metrics to use in this command can be limited with the `--targets` option.
```

Comment:
`metrics files to analyze` 

remove `this command` - it's kinda obvious

Reply:
not sure how useful this paragraph is considering that we have exactly the same info below

Reply:
I think `--targets` is an option we want to emphasize so it's worth mentioning in the description, similar to the paragraph a bout `targets` in the description of https://dvc.org/doc/command-reference/status. I reworded the p and moved it up now, please check.

Reply:
in this case all this is so close to the option and it has so obvious information (option like dry description) that I still don't understand the reason to include this tbh

Reply:
> this is so close to the option

I had moved the paragraph up already to be the 2nd paragraph. But OK, removed it in f0f8c2db.